shared/toks-components/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { theme } from '@depromeet/theme';
+import styled from '@emotion/styled';
+import { Flex } from '@toss/emotion-utils';
+import React, { Dispatch, ReactNode, SetStateAction } from 'react';
+
+import { Text } from '@depromeet/toks-components';
+
+interface Props {
+  modalTitle: string;
+  contents: ReactNode;
+  button: ReactNode;
+  setIsModalOpened: Dispatch<SetStateAction<boolean>>;
+}
+export function Modal({ modalTitle, contents, button, setIsModalOpened }: Props) {
+  const onClickOutsideModal = () => {
+    setIsModalOpened(false);
+  };
```

Comment:
밑에 
```suggestion
  const onClickOutsideModal = () => {
    onClose?.()
  };
```
식의 코드가 작성되야 할 것 같아요